demos/dropdown/cyclow/es6/src/dropdown.js

```diff
@@ -0,0 +1,24 @@
+import {Block} from 'cyclow'
+
+import {goods} from '../../../goods.js'
+
+const Dropdown = () => Block({
+  on: {
+    'in.init': () => state => ({goods, chosen: goods[0]}),
+    'dom.select': index => state => ({...state, chosen: goods[index]})
+  },
+  view: ({goods, chosen}) => ({
+    attrs: {id: 'app', class: chosen.type},
+    content: [
+      'I postit that ',
+      {
+        tag: 'select',
+        on: {change: (e, next) => next({select: e.selectedIndex})},
```

Comment:
On debugging it seems that `e.selectedIndex` is undefined, the event object has no such property.

Which means that on line 8 the `index` is undefined, causing `goods[index]` to be undefined too, which finally causes `chosen` to be undefined on line 11.

Is Cyclow supposed to augment the change event on the dropdown with the `.selectedIndex` prop?